src/main/java/com/google/cloud/tools/maven/CloudSdkAppEngineFactory.java

```diff
@@ -258,9 +263,16 @@ public void onOutputLine(String line) {
 
     private final PrintStream logFilePrinter;
 
-    FileOutputLineListener(File logFile) {
+    FileOutputLineListener(final File logFile) {
       try {
         logFilePrinter = new PrintStream(logFile, StandardCharsets.UTF_8.name());
+        Runtime.getRuntime()
```

Comment:
This is probably fine, but I was referring to adding an `exitListener` that would close this `PrintStream` on like L170. Would require getting the PrintStream from this object though.